todo-app/src/storage.js

```diff
@@ -0,0 +1,8 @@
+export function loadTodos() {
```

Comment:
This should also be removed - you are persisting data via the API, not localStorage.